trace/trace.go

```diff
@@ -100,6 +100,13 @@ func WithSpan(parent context.Context, s *Span) context.Context {
 	return context.WithValue(parent, contextKey{}, s)
 }
 
+// All available span kinds.
+const (
+	SpanKindUnspecified = iota
```

Comment:
Perfect, I was just about to comment on this, on the previous change to ask for this change :)